styles.css

```diff
@@ -266,43 +500,84 @@ header {
     height: 100px;
   }
 
-  .first_img {
-    background-image: url("./assets/first_img1.png");
-    position: absolute;
-    background-repeat: no-repeat;
-    top: 100px;
-    right: 0;
-    width: 100px;
-    height: 100px;
+  .images {
+    width: 100%;
   }
 
-  .second_img {
-    background-image: url("./assets/img_twp.png");
-    position: absolute;
-    background-repeat: no-repeat;
-    top: 130px;
-    right: 150px;
-    width: 100px;
-    height: 100px;
+  .ruby {
+    display: none;
   }
 
-  .img_three {
-    background-image: url("./assets/img_three.png");
-    background-repeat: no-repeat;
-    position: absolute;
-    top: 500px;
-    left: 260px;
-    width: 100px;
-    height: 100px;
+  .works {
+    display: flex;
+    flex-direction: column;
+    row-gap: 6rem;
```

Comment:
- [ ] [OPTIONAL] Great job 👍 . Please make sure you use the right background color in your design as it is good practice.

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/176549068-96317601-be0a-4f6d-91bb-6e1988c41280.png)  |  ![iUser's design](https://user-images.githubusercontent.com/74814780/176548881-60b5ce05-4480-4b45-b788-4a3e54b174f0.png)